libs/filamat/CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ include_directories(${CMAKE_BINARY_DIR})
 add_library(${TARGET} STATIC ${HDRS} ${PRIVATE_HDRS} ${SRCS})
 target_include_directories(${TARGET} PUBLIC ${PUBLIC_HDR_DIR})
 
-target_link_libraries(${TARGET} shaders filabridge backend utils smol-v)
+target_link_libraries(${TARGET} shaders filabridge utils smol-v)
```

Comment:
You can also remove `backend` from filamat-android/CMakeLists.txt.

Reply:
Done